pallets/runtime/src/mips.rs

```diff
@@ -76,6 +76,8 @@ pub struct MipsMetadata<BlockNumber: Parameter, Hash: Parameter> {
     end: BlockNumber,
     /// The proposal being voted on.
     proposal_hash: Hash,
+    /// The proposal url for proposal discussion.
+    url: Option<Vec<u8>>,
```

Comment:
We likely want to introduce a wrapper similar to `DocumentUri` here instead of the simple type `Vec<u8>`. See #212 and #223.

Reply:
I am working on it now